packages/compass-schema-validation/src/components/document-preview/document-preview.jsx

```diff
@@ -40,6 +46,7 @@ class DocumentPreview extends Component {
   static propTypes = {
     document: PropTypes.object,
     isLoading: PropTypes.bool,
+    loadSampleDocument: PropTypes.func
```

Comment:
Small nit: react callback properties usually named after what element event being emitted, not as the action that will be called, in this case this might rather be `onLoadSampleClick`. We don't have this followed well across the whole codebase, but really trying to stick to it for the new code we are adding.

Reply:
Well that makes sense. Thanks for reminding, will do the needful 👍